css.css

```diff
@@ -0,0 +1,125 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+body {
+    font-family: 'Roboto', sans-serif;
+}
+                                            /*topnav*/
+.topnav {
+    float: right;
+    border-left: solid 3px #f9f9f9;
+    border-bottom: solid 3px #f9f9f9;
+    width: 95%;
+    display: flex;
+    align-items: center;
+}
+.topnav__search {
+    margin-left: 10px;
+}
+.topnav__search__input {
+    border: hidden;
+}
+.topnav__search__input:focus {
+    outline: none;
```

Comment:
removing focus is bad for accessibility (try to navigate through the site with keyboard only)